00-boilerplate/src/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { App } from "./app";
+
+ReactDOM.render(
+  <div>
+    <App />
+  </div>,
+  document.getElementById("root")
+);
```

Comment:
If you open the console you will get a warning in react 18 you should not use this approach, will send you an updated example

Reply:
done